### PR TITLE
Add storybook-addon-a11y definitions

### PR DESCRIPTION
**Pull request by pascalduez**
See https://github.com/storybooks/storybook/tree/master/addons/a11y

### CODE REVIEW

File: definitions/npm/@storybook/addon-a11y_v3.x.x/test_addon-a11y_v3.x.x.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+// @flow
+
+import { describe, it } from 'flow-typed-test';
+import React from 'react';
+
+// $ExpectError
```
Comment (AndrewSouthpaw) [inline definitions/npm/@storybook/addon-a11y_v3.x.x/test_addon-a11y_v3.x.x.js]: What's this for?

Comment (pascalduez) [reply definitions/npm/@storybook/addon-a11y_v3.x.x/test_addon-a11y_v3.x.x.js]: Without the comment:
```
Cannot resolve module `@storybook/react`.

import { storiesOf } from '@storybook/react';
                           ^^^^^^^^^^^^^^^^^^
```

Without the import:
```
Cannot resolve name `storiesOf`.

storiesOf('button', module)
^^^^^^^^^
```

Choose your poison :)

Comment (pascalduez) [reply definitions/npm/@storybook/addon-a11y_v3.x.x/test_addon-a11y_v3.x.x.js]: Look above ;)

Comment (pascalduez) [reply definitions/npm/@storybook/addon-a11y_v3.x.x/test_addon-a11y_v3.x.x.js]: So what's the desired update then? @AndrewSouthpaw @GAntoine

**Comment by pascalduez** [comment]
@AndrewSouthpaw @GAntoine "changes-requested" but which one? :)

**Comment by GAntoine** [comment]
This looks good to me, I'll go ahead and merge it in.

**Comment by AndrewSouthpaw** [comment]
👍

**Pull request merged**